com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicStreamPlayer.cs

```diff
@@ -140,14 +160,22 @@ internal bool LoadFromFile(string newPath)
             return InitializeAfterLoad();
         }
 
+        /// <summary>
+        /// Close and reopen the Alembic stream. Use this to apply the new stream settings.
```

Comment:
Suggestion:
`Closes and reopens the Alembic stream. Use this method to apply the new stream settings.`